commands/commands.go

```diff
@@ -226,6 +226,7 @@ Make sure "audio_play_cmd" is set to a program that can play wav files.
 }
 
 func (cmd Dale) Run() gribble.Value {
+	return nil
```

Comment:
Perhaps add a comment about why you nerfed this function? I see the related `# audio_play_cmd` and possibly `AudioProgram:    "", //"aplay",` so just curious why you also had to return early here.

Reply:
it has to die. He says it's an easter egg and I see no need of such things. This whole pile of code needs a lot of love